frontend/simple/views/CreateGroup.vue

```diff
@@ -66,7 +66,7 @@
                     <i class="fa fa-usd" aria-hidden="true" ></i>
                   </span>
                 </p>
-                <span v-if="errors.has('incomeProvided')" data-control="incomeProvided" class="help is-danger">The Income Provided field must be a numeric currency amount and may contain 2 decimal points.</span>
+                <span v-if="errors.has('incomeProvided')" data-control="incomeProvided" class="help is-danger"><i18n>The Income Provided field must be a numeric currency amount and may contain 2 decimal points.</i18n></span>
```

Comment:
OK, so on all of these above what ends up generated is `<span><span> .. </span></span>`.

Can you, knowing that, make it work just with the `<i18n>` tag? E.g. so that all of these (and elsewhere in other files) looks like this:

```html
<i18n v-if="errors.has('incomeProvided')" data-control="incomeProvided" class="help is-danger">The Income Provided field must be a numeric currency amount and may contain 2 decimal points.</i18n>
```